src/main/resources/data/minecraft/recipes/stone_shovel.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This will spam errors in the console. Also, please make the vanilla recipe conditional and add a config for it.. Same applies for all of your replaced recipes